src/others/vg_lite_tvg/vg_lite.h

```diff
@@ -589,6 +589,7 @@ typedef unsigned int        vg_lite_color_t;
     typedef enum vg_lite_param_type
     {
         VG_LITE_SCISSOR_RECT,                   /*! count must be 4n for x, y, right, bottom */
+        VG_LITE_GPU_IDLE_STATE,                 /*! 0: busy, 1: idle */
```

Comment:
May `VG_LITE_PARAM_GPU_IDLE_STATE` be better?

Reply:
This is the naming of the official driver, keep it consistent here.